ios/engine/KMEI/KeymanEngine/Classes/LanguagePicker/LexicalModelPickerViewController.swift

```diff
@@ -419,13 +417,30 @@ class LexicalModelPickerViewController: UITableViewController, UIAlertViewDelega
   }
   
   func showAddLexicalModel() {
-    let button: UIButton? = (navigationController?.toolbar?.viewWithTag(toolbarButtonTag) as? UIButton)
-    button?.isEnabled = false
-    let vc = LanguageViewController(Manager.shared.apiLexicalModelRepository) //may need to be different for models
-    navigationController?.pushViewController(vc, animated: true)
-    if language == nil {
-      setIsDoneButtonEnabled(true)
```

Comment:
No love for the `setIsDoneButtonEnabled(true)` bit?  I see that it's still in use for some other cases (like after updating).

Reply:
It's because of the "if language == nil {" wrapper portion. This started out as a copy-search-replace of the KeyboardPickerViewController, but there is no generic, pick a language first, then pick lexical model, since you only launch this picker from the Language's list of lexical models.

so I made language *alway* there:
`  public var language: Language!
`
Any time it was nil now cannot happen. But I see your point, now I should probably just enable the done button where it was conditional.